client/app/page.tsx

```diff
@@ -24,8 +24,12 @@ export default function Home() {
           <Heading
             title={houseName}
             buttonComponent={
-              <Button variant={'outline'} onClick={() => router.push('/rules')}>
-                Rules
+              <Button
+                variant={'outline'}
+                className='ml-3'
+                onClick={() => router.push('/house-info')}
+              >
+                House Info
```

Comment:
Better to add `align-Items:center` for the title and the right button!
<img width="479" alt="Screenshot 2024-04-15 at 1 45 22 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/b6fa3993-6e8e-48cc-abff-b1ea1be645af">

Reply:
Added it.
553dcf24bb4ebd09d9f85ed231d6e37376a6d457